hack/Containerfile.rpm-builder

```diff
@@ -0,0 +1,12 @@
+FROM registry.ci.openshift.org/openshift/release:rhel-8-release-golang-1.19-openshift-4.13
+
+RUN rm -rfv /etc/yum.repos.d/ci-rpm-mirrors.repo /etc/yum.repos.d/localdev* && \
+    dnf install \
+        --setopt=tsflags=nodocs \
+        --setopt=install_weak_deps=False \
+        --repo=rhel-8-for-x86_64-baseos-rpms \
```

Comment:
We're supporting RHEL 9 now in the main branch. Can we use RHEL9 containers / dependencies?

Reply:
This is because we use `registry.ci.openshift.org/openshift/release:rhel-8-release-golang-1.19-openshift-4.13`.
I see that `rhel-9-release-golang-1.19-openshift-4.13` exists but it fails due to some certificates for `rhel-9-for-x86_64-baseos-rpms` repo.
Also, we're still using `rhel-8-release-golang-1.19-openshift-4.13` in the CI jobs